### DISCUSSION TITLE
Keep founder benefits indefinitely

### DISCUSSION THREAD

**Discussion by clone1018** [Glimesh/company#14] [Ideas]
Pooh in Discord had the idea that users should keep the founder benefits on the site even if they decide to cancel their recurring Glimesh Platform Founder subscription. 

For the discussion, here are the current benefits:

**Glimesh Supporter**
 * Colored Avatar Ring in all chats
 * Allowed usage of global animated emotes

**Glimesh Founder**
 * All Supporter Perks
 * Gold Username Color
 * Animated Avatar Ring

**Comment by Paco1342** [comment]
The Founder's Sub was created as a way to both allow people to support the platform above the normal platform sub, and to thank them for that support. Part of that thanks is providing the "Founder's Ring" on the avatar and eventually a badge on their profile. Seeing as the Founder's Sub is going to be a "promotional" item and will at some point no longer be available, it seems counterintuitive to take away those cosmetic perks without an active Founder's Subscription. Any of the "shared perks" with the standard platform sub, such as animated emotes, should expire as normal with no subscription or carry on with a standard platform sub.

**Reply by Nekron297** [reply]
What im reading is that it shuld just be the cosmetic perks that carry through cancellation, correct?

**Reply by Paco1342** [reply]
Correct

**Comment by Spectacularpooh2414** [comment]
That is exactly what this is about..

Look at it this way people: with the current system in place with the founders benefits, say you was actually paying for it monthly.. but then it would just so happen that one day you can't spend it for $25 anymore, that or you need the money for something more important in life..  at some point you have to look out for your own plus you have done enough supporting on the site as it is with just that one purchase of $25 because its a pretty heavy investment in the long run..

You should be rewarded as such by allowing you to keep what you bought as a founder to recognize your status, you shouldn't have to spend anymore..

if its a "founders" perk then let it be as such where if you sat up there and spent the necessary money to have this status then you should keep them so everyone knows you bought into this system.. An Idea i propose is you have a sub tier list where it'd be like the following: 

Tier 1 - $5.00
Tier 2 - $10.00
Tier 3 - $15.00 
(seeing as this is a new site, having it go by 5s would be respectable and Glimesh as a website would still benefit)

Founders - $25.00 (would be a one time purchase)

**Reply by clone1018** [reply]
I understand your opinion on this, but from my perspective giving permanent perks for a recurring product will just encourage everyone to subscribe to it for one month and then immediately drop it. This won't make sense for the business because the entire point of Platform Subscriptions is to build direct recurring revenue. 

The founder subscription is intended to be a heavy investment and the perks are intended to be minimal to encourage those who do not have the money, or do not want to spend the money to not be required to subscribe to it. The founder subscription for Glimesh is intended to be an option for those who want to donate money to the platform regardless of the perks.

My questions for you are:
1. What would prevent everyone from subscribing and then immediately unsubscribing the next month? 
2. How would the business create an attractive subscription model for it's customers without taking away the benefit if the user lets their subscription lapse for whatever reason?

For the discussion, here are the current benefits:


**Glimesh Supporter**
 * Colored Avatar Ring in all chats
 * Allowed usage of global animated emotes

**Glimesh Founder**
 * All Supporter Perks
 * Gold Username Color
 * Animated Avatar Ring

**Reply by Spectacularpooh2414** [reply]
I think this i can answer

1. for what i just mentioned earlier, if the prices for this are exactly as it is for $25 literally that alone is going to make people make that investment to watch more streamers and continue to support the product, as for what one streamer just said: He'd have no control over his $25 monthly subscription especially since he has so many others he wants to support, on top of that it can easily come at any point where that one person will feel like they are being forced to keep the founder benefits and they'd leave right then and there because $25 is way too much really, its limits what they do in life and it limits the amount of support streamers can have for one another.

one other option you can have is you'd raise the price more, like at least raise it up to $35 because that would tell you right then and there how committed the person would be to the site to support it and its streamers.. a monthly founder subscription is honestly not fair that one person that could stop for whatever reason the next month and it'll never be really shown that they put the work in.. THAT alone would be enough for people that want to help the site not stick around.. you are taking away interest like that. i shouldn't be punished for going cheap to support you with a $5 or $10 sub, im still helping you.

2. the prices i just mentioned with the tier sub lists would actually help because if you take a look at how Twitch or even Dlive handle subscriptions it is WAAYY insane with how its handled so have it be like this would make streamers want to continue supporting the product more... just being slightly cheaper would be an attractive model for people to look at.

**Comment by ScottKemosobe** [comment]
I don't know what the complete differences are between the normal and founders subs, but you could always just make the Founder's perks be a 1-time purchase. Could also maybe ask a little more to make it something more indicative of you are trying to help the site get off the ground more than just wanting to look "cool" to other people (which I know some are 100% going to do with it). Or could make it where you need to sub for X months before you keep it permanently.

I also personally do not entirely like the concept of where you only get to show people you were there from the start unless you pay for it. (Which is part of the reason I said maybe raise the price to have it mean more than just I was here early) Which is why I thought maybe you can also always convert the permanent Founder's thing into a badge for anyone who has an account. Like an "Alpha Glimmer" badge or something that you only get if you have an account before some date (maybe before Beta and then people get a "Beta Glimmer" badge)

**Comment by tjcasey** [comment]
I think the 'founder' title is what people are after here and they don't want to pay monthly (or at all) for it.  They want to have a way to show they were here for the launch.  This could be shown by a simple profile badge?  Possibly other badges for helping report bugs during alpha etc.

For the recurring monthly platform sub, perhaps the name of the product could be changed to Glimesh Pro/Plus/Mega/Super Dooper etc.   so it better reflects it's purpose?   Also, I'd expect if I'm paying a monthly charge of $25USD that the perks I receive are above those received by any one time purchase.  There needs to be a reason for people to continue the platform sub monthly.

**Comment by Cykotiq** [comment]
I agree 100% with clone. The founder perks should be minimal as its targeted at those who want to financially support glimesh more heavily. users should not being doing it for the perks, and it's good that the perks are minimal.

Those who ARE doing it just for the perks/bragging rights should be gently discouraged by having it be a recurring month-to-month payment. If we were to make it a one-time payment, or have it persist if/when the founders roles is end-of-lifed, then those supporters have no reason to continue supporting glimesh with a normal platform sub because they'd already have all the perks forever

**Reply by Cykotiq** [reply]
if recognition for being there since the beginning is what people are after, than an OG glimmer badge/achievement would be more suitable. I know the beam OG badge was greatly appreciated by those who had it after mixer blew up. It could be a highly coveted badge/achievement that only those who were there since the beginning would have.

**Comment by Heiwa1580** [comment]
I have long thoughts on this just need to find the time to write them.

**Reply by Heiwa1580** [reply]
So I think this whole idea has two realms of thought. There is the business aspect and the mindshare aspect. 

On the business side of things I think we must establish a few reasons to have subscriptions. I think that subscriptions will help in the beginning a buffer zone of finances while we establish an income through streamer subs. Eventually this idea of "founder subs" will have to evolve to something else. But on the business side of thing the goal of this is to create a stable income while we don't have subs income coming from people subbing to streamers.

On the mindshare aspect we must also realize that these subscriptions and the benefits tied to them carry some value in the mind of our consumers. The mindshare of being a founder supporter and having the benefits revolving around that have a monetary value to people. 

I think we need to find the sweet spot between those two things and how do we make sure that both the business and the mindshare are properly compensated to keep both consistent and sustainable. 

I think our current 25$ a month setup is going to be a serious commitment and I believe there will be people who will jump onto that. But I think that there is a desire and a need to fulfill in mindshare of being an OG to Glimesh. I think the current system where benefits drop out completely does not satisfy mindshare and once the newness of Glimesh wears off the monthly package will feel worthless after losing whatever original cosmetics you had to show off. 

I also think that Microtransactions have shown great success in industries to where they seem to be the going product for many sites, games, and other products. My proposal is to have a few different options to support Glimesh and to also help establish an income, while satisfying mindshare.

### Supporter Only

This would be a three tiered option and subscribing to this would be to just support the platform. I think that this could be shown by a small icon per tier on profiles. Think like a Bronze, Silver, and Gold Glimdrop on the profile page where the socials would be. These would be a 5. 15, 25$ sub options and would just come with that icon and would also come with a discount on Packs. This would leave support as support and not attached to the benefits alone.

### Founder's Pack

This would be a one time purchase and would be exclusive to purchase within the first X number of months. These would have tiers as well under a one time purchase. 10$ would get you just the Gold username, 20$ would get you the Gold ring and gold username and 25$ would be the animated emotes (limit 3-5) plus gold stuff.

I think periodically time to time we can make more of these packs to sell as specials revolving around events and what not. This could become a more consistent source of income while also giving us future opportunity for more. It would also satisfy the mindshare of being a founder. These packs could also include other benefits but I think that this would be a way to satisfy the mindshare of being a founder while also giving us two options for creating income for the platform.

### Emotes and Cosmetics

The last thing I think we could do to add value to the packs would be to charge for premium cosmetics and emotes. I think being able to unlock any animated emote for 5$ would make the 25$ pack more valuable as you get exclusive benefits plus the value of the emotes. But this would also allow someone to just purchase the emotes that they want to use. 

I am sure there could be tweaking but I think doing something like this would add value to both the business and the mindshare while also getting us future options for more Glimdrop money grabs.

**Reply by clone1018** [reply]
My initial quick thoughts are this seems like a very gamified system, which would lead to increased profits and addiction from our users, but what we need to consider is: Is this model right for Glimesh?

Platform Subscriptions are intended as a way to support the platform monthly through recurring subscriptions, and we throw in some perks so you can show off. The original model is simple and we include all of the "features" with the $5/mo package, so the founder package is literally just extra.

**Reply by Heiwa1580** [reply]
I see it less like a gamified system to be more of a limited merch release type of system. Something similar to how Supreme operates https://www.supremenewyork.com. 

I do think that there are pros and cons to this and it may not be the right method for Glimesh but I think that becomes a whole different discussion. I also think in the scope for launch it may be an overly complex system to code for launch.

My main goal was to provide a way to allow users to support the platform while also still receiving limited benefits for being early adopters. But also leave an open option to ongoing monthly support with minimum benefits. But I think we could accomplish the same by simply renaming the Founder aspect as Moltenllama suggested as well.

**Reply by ScottKemosobe** [reply]
My initial thoughts reading that are that it feels like I am reading an EA or Activision game development discussion about how much you can rip apart things to sell back for profit. Not trying to say good or bad, just that is the feel I got.

You say microtransactions have proven success in the industry, but they have also proven to be something VERY divisive and most people tend to be less "accepting" of them and more just not caring about them existing with most others seeming to outright hate them in most situations. I feel like if people come in and see a Glimesh "cash shop" style system where you can pay for emotes, special tags, and other things, they will probably see it more as a cash grab and just leave back to where they already have a home, of sorts, especially since most of those things are free elsewhere. Yes, none of those things are needed, but it is more optics and how people will perceive it.

**Reply by Heiwa1580** [reply]
Yeah I agree mindshare is important and valuable and once you lose it is very difficult to get it back. Whatever we do decide I think optics will be very important to get right.
